runtime/runtime-params-estimator/src/cost.rs

```diff
@@ -241,6 +241,13 @@ pub enum Cost {
     /// produces the steepest line.
     ContractCompileBaseV2,
     ContractCompileBytesV2,
+    /// The cost of contract deployment per byte, without the compilation cost.
+    ///
+    /// Estimation: Measure the cost of two contract that have the same code but
+    /// include different data sections. One data section is significantly
+    /// larger. The cost difference is pure overhead of moving around bytes that
+    /// are not code. Divide this cost by the difference of bytes.
+    DeployBytes,
```

Comment:
Ok, yeah I can create such a contract and it should be more accurate.

Second observations makes sense to me. The way we measure it, what we call "compilation cost" includes this additional IO produced by code and the "pure deployment" cost does not. I guess that makes the estimation of `DeployBytes` a bit inaccurate. I can think of a few ways to improve this but none of them are particularly simple. And since we only use this to get a feeling for which fraction of the deployment cost is compilation, I'm not sure if it's worth it to spend the time on that. WDYT?